dev/pyproject.py

```diff
@@ -147,6 +142,13 @@ def build(package_type: PackageType) -> None:
                     "boto3>1",
                     "botocore",
                 ],
+                "mlserver": [
+                    # Required to serve models through MLServer
+                    # NOTE: remove the upper version pin once protobuf is no longer pinned in
```

Comment:
nit: I think we can remove this note now?

Reply:
ah yeah, let's remove it.